### PR TITLE
Drop EoL Ubuntu 16.04

### PR DESCRIPTION
**Pull request by root-expert**
#### Pull Request (PR) description
Ubuntu 16.04 has been EoL since 30/04/21

### CODE REVIEW

File: manifests/web.pp [hunk 1]
```diff
@@ -151,7 +151,7 @@
   # zabbix frontend 5.x is not supported, among others, on stretch and xenial.
   # https://www.zabbix.com/documentation/current/manual/installation/frontend/frontend_on_debian
   if $facts['os']['name'] in ['ubuntu', 'debian'] and versioncmp($zabbix_version, '5') >= 0 {
-    if versioncmp($facts['os']['release']['major'], '16.04') == 0 or versioncmp($facts['os']['release']['major'], '9') == 0 {
+    if versioncmp($facts['os']['release']['major'], '9') == 0 {
```
Comment (ekohl) [inline manifests/web.pp:154]: Should it also drop Debian 9, which we also consider unsupported in Vox Pupuli modules.

File: manifests/web.pp [hunk 2]
```diff
@@ -231,7 +231,7 @@
       # Check OS release for proper prefix
       case $facts['os']['name'] {
         'Ubuntu': {
-          if versioncmp($facts['os']['release']['major'], '16.04') >= 0 {
+          if versioncmp($facts['os']['release']['major'], '18.04') >= 0 {
```
Comment (ekohl) [inline manifests/web.pp:234]: I think this whole condition can be dropped. You should never get to the `else` branch. Similar for Debian below. The default branch can't be reached so the whole case can be replaced with `$php_db_package = "php-${db}"`.

**Review by ekohl — CHANGES_REQUESTED** [review]

File: manifests/web.pp (hunk 2 above)
Comment (bastelfreak) [reply manifests/web.pp]: cleaned it up

File: manifests/web.pp (hunk 1 above)
Comment (bastelfreak) [reply manifests/web.pp:154]: I did like to drop it on another PR

**Pull request merged**